lightly_bucket/.lightly/predictions/object_detection_sportradar/schema.json

```diff
@@ -0,0 +1 @@
+{"task_type": "object-detection", "categories": [{"id": 0, "name": "human", "supercategory": "none"}]}
```

Comment:
Also here, try to dump this using `indent=4` for nicer/ more human readable formatting :)